src/api/headset.c

```diff
@@ -292,6 +293,18 @@ static int l_lovrHeadsetUpdate(lua_State* L) {
   return 0;
 }
 
+static int l_lovrHeadsetGetMirrorTexture(lua_State* L) {
+  Texture *texture = NULL;
+  if (lovrHeadsetDriver->getMirrorTexture)
+    texture = lovrHeadsetDriver->getMirrorTexture();
+  if (texture)
+    luax_pushobject(L, texture);
+  else
+    lua_pushnil(L);
```

Comment:
Small nit, `luax_pushobject` handles NULL and will push nil in that case, so the `if` here isn't really necessary.